core/Config.php

```diff
@@ -331,6 +331,26 @@ public function &__get($name)
         return $section;
     }
 
+    /**
+     * Returns a boolean variable setting for convenience
+     * when calling e.g.
+     * Config::getInstance()->getBool('General', 'force_ssl')
+     * This also documents that a boolean is only true iff
+     * it is equal to 1.
+     *
+     * @param string $section Configuration section
+     * @param string $name variable name
+     * @return bool whether it is considered set true (== 1)
+     *
+     * @api
+     */
+    public function getBool(string $section, string $name): bool
+    {
+        $section = $this->$section;
+        $value = $section[$name] ?? 0;
+        return $value == 1;
```

Comment:
To be fully sure the value was configured as boolish value we could harden the check to something like:
```suggestion
        return 1 === $value || '1' === $value || true === $value;
```
Otherwise value like `1test` might be accepted, as it converts to `1` when casted to int.

Reply:
Left the check as is for now as it seems correct also for the cases you mentioned above, @sgiehl . Left if because of BC. I think we can implement stricter tests in the next version. I left a comment in that regard.